cfgov/v1/__init__.py

```diff
@@ -16,37 +18,53 @@
 )
 from django.utils.module_loading import import_string
 from django.utils.timezone import template_localtime
+from django.utils.translation import ugettext, ungettext
 
 from jinja2 import contextfunction, Markup
-from sheerlike import environment as sheerlike_environment
 from compressor.contrib.jinja2ext import CompressorExtension
-from flags.template_functions import flag_enabled, flag_disabled
-from v1.util.util import get_unique_id
 
 from wagtail.wagtailcore.rich_text import expand_db_html, RichText
+
 from bs4 import BeautifulSoup, NavigableString
-from processors.processors_common import fix_link
+
+from flags.template_functions import flag_enabled, flag_disabled
+
 from core.utils import signed_redirect, unsigned_redirect
-from v1.routing import get_protected_url
+from processors.processors_common import fix_link
+from sheerlike import environment as sheerlike_environment
```

Comment:
I'm not following the ordering scheme you're now using for the imports in this file. Did you do part of it and not finish?

Reply:
This file is somewhat beyond hope, IMHO. I'll see what running `isort` does for it.